packages/strapi-design-system/src/v2/MainNav/NavCondense.tsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 
 import { ChevronRight, ChevronLeft } from '@strapi/icons';
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 import { useMainNav } from './MainNavContext';
 import { Icon } from '../../Icon';
 import { VisuallyHidden } from '../../VisuallyHidden';
 
-const NavCondenseWrapper = styled.button`
+const NavCondenseWrapper = styled.button<{ condensed: boolean }>`
```

Comment:
Can we please refactor this to use `Flex`? A lot of this css could be moved to props instead imo.